OWML.Launcher/SocketListener.cs

```diff
@@ -0,0 +1,88 @@
+﻿using OWML.Common;
+using System;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OWML.Launcher
+{
+    public class SocketListener
+    {
+        private static int _port;
+
+        public SocketListener(int port)
+        {
+            _port = port;
+            new Task(SetupListener).Start();
+        }
+
+        private void SetupListener()
+        {
+            TcpListener server = null;
+            try
+            {
+                IPAddress localAddr = IPAddress.Parse("127.0.0.1");
+
+                server = new TcpListener(localAddr, _port);
+
+                server.Start();
+
+                Byte[] bytes = new Byte[256];
```

Comment:
I think expanding it to 1024 or so might be good idea, stack traces could be big